vagga.yaml

```diff
@@ -21,8 +21,8 @@ containers:
             - !Env PATH: /bin:/sbin:/usr/bin:/usr/sbin:/usr/local/bin:/usr/local/sbin:/work/.vagga/nightly-home/.cargo/bin
             - !Sh rustup update
             - !Sh cargo install -f cargo-update
-            - !Sh cargo install -f clippy
-            - !Sh cargo install rustfmt || cargo install-update rustfmt
+            - !Sh cargo install-update clippy
```

Comment:
I have encountered some cases, where no new clippy version was released, but because of its linkage to rustc, I had to rebuild clippy (no new version) after updating rustc. Is there a specific reason for this change?